Userland/Libraries/LibWeb/FileAPI/File.cpp

```diff
@@ -24,7 +24,7 @@ File::~File() = default;
 WebIDL::ExceptionOr<JS::NonnullGCPtr<File>> File::create(JS::Realm& realm, Vector<BlobPart> const& file_bits, DeprecatedString const& file_name, Optional<FilePropertyBag> const& options)
 {
     // 1. Let bytes be the result of processing blob parts given fileBits and options.
-    auto bytes = TRY_OR_RETURN_OOM(realm, process_blob_parts(file_bits, static_cast<Optional<BlobPropertyBag> const&>(*options)));
+    auto bytes = TRY_OR_RETURN_OOM(realm, process_blob_parts(file_bits, options.has_value() ? static_cast<Optional<BlobPropertyBag> const&>(*options) : BlobPropertyBag {}));
```

Comment:
This still seems wrong - `options` is an `Optional<FilePropertyBag> const&`, so `*options` is an `FilePropertyBag const&`, and `static_cast`ing that to `Optional<BlobPropertyBag> const&` again seems like nonsense?

Reply:
Yeah, this whole thing was a mess. Should be better now!